src/Twig/Extension/FormatterMediaExtension.php

```diff
@@ -105,26 +124,42 @@ public function path($media, $format)
         return $this->getTwigExtension()->path($media, $format);
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     */
     public function getNodeVisitors()
     {
         return $this->getTwigExtension()->getNodeVisitors();
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     */
     public function getFilters()
     {
         return $this->getTwigExtension()->getFilters();
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     */
     public function getTests()
     {
         return $this->getTwigExtension()->getTests();
     }
 
     public function getFunctions()
     {
-        return $this->getTwigExtension()->getFunctions();
+        return [
+            new TwigFunction('sonata_media', [MediaRuntime::class, 'media']),
+            new TwigFunction('sonata_thumbnail', [MediaRuntime::class, 'thumbnail']),
+            new TwigFunction('sonata_path', [MediaRuntime::class, 'path']),
```

Comment:
this is my proposal about names of new functions, but we could remove the 'sonata_' if you prefer it. I was trying to avoid colisions with possible user defined functions.

wdyt @VincentLanglet ?

Reply:
I think it's ok to have those prefix for the reason you gave (and for context). And I already saw method with the sonata prefix in some other bundles.